netpyne_ui/netpyne_geppetto.py

```diff
@@ -180,13 +185,7 @@ def stopExperiment(self, experiment_name):
         }
 
     def find_tutorials(self):
-        only_files = [f for f in os.listdir(constants.NETPYNE_WORKDIR_PATH) if
-                      os.path.isfile(os.path.join(constants.NETPYNE_WORKDIR_PATH, f))]
-
-        def _filter(_file):
-            return '.py' in _file and 'tut' in _file and 'gui' in _file
-
-        return list(filter(_filter, only_files))
+        return glob(join(NETPYNE_WORKDIR_PATH, "**/gui_tut*.py"), recursive=True)
```

Comment:
this is the place where we load tutorials from a path scanning @aranega

Reply:
Thanks for the details! It's way more clear for me :)